cljfmt/src/cljfmt/core.cljc

```diff
@@ -280,10 +280,22 @@
     (or (symbol-matches-key? (fully-qualified-symbol possible-sym context) key)
         (symbol-matches-key? (remove-namespace possible-sym) key))))
 
+(defn- parent-is-thread-first? [zloc context]
+  (some #(form-matches-key? zloc % context) #{'-> 'as-> 'some-> 'cond->}))
```

Comment:
A better name for this would be: `form-matches-thread-macro?`. The list of threading symbols should also be configurable based on the context, and I think this might need to be refactored so we only call `form-symbol` once.